spec/models/note_spec.rb

```diff
@@ -388,8 +388,8 @@
   end
 
   describe '#feature' do
-    Setting['advanced.instructions_feature_first'] = '__FEATURE_FIRST'
-    Setting['advanced.instructions_feature_last'] = '__FEATURE_LAST'
+    ENV['instructions_feature_first'] = '__FEATURE_FIRST'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.